### ISSUE TITLE
Error loading LibStub

### ISSUE THREAD

**Issue by Jakobud** [mitchnull/Bazooka#4]
Errors loading LibStub for both Wow Classic WOTLK and also for WoW Classic Era/Season of Discovery

```
Bazooka/libs/LibJostle-3.0/lib.xml:4 Error loading Bazooka/LibStub/LibStub.lua
```

**Comment by Jakobud** [comment]
I have investigated this further. There are definitely some problems with how LibStub is being loaded and it affects LibJostle and essentially makes it not work:

![image](https://github.com/mitchnull/Bazooka/assets/87356/a2c7af53-aba5-4b3a-a5f1-de8ef27156d1)

I noticed here there are 2 different versions of 3.0.7:

https://www.wowace.com/projects/bazooka/files

I found that the latest version that is `187.31 KB` does NOT work. However the other version that is `189.07 KB` DOES WORK fine and does not have the LibStub issue.

I used WinMerge to compare the two different "versions" of 3.0.7 and found only that the files appear to be identical however the version that is broken is using UNIX end-of-line returns ("\n") while the version that works is using Windows end-of-line returns ("\r\n"). I can't imagine why this would make a difference but apparently it does make a difference. I replaced my addon with the "working" version in Wow Classic Era and WoW classic WOTLK and now it works fine.

Also the only other differences between the two "versions" is that the "working" version has a CHANGES.txt and this was replaced by a CHANGELOG.md in the "broken" version. Obviously this doesn't make a difference for this issue.
